README.md

```diff
@@ -1,2 +1,40 @@
-# terraform-provider-flux
-Terraform provider for Flux v2
+# Terraform provider FluxV2
```

Comment:
Please use the repo name here `terraform-provider-flux`

Reply:
done